spark-3.1-spanner-lib/src/test/resources/db/populate_ddl.sql

```diff
@@ -48,7 +48,20 @@ CREATE TABLE ATable(
 ) PRIMARY KEY(A);
 
 CREATE TABLE compositeTable (
-  id STRING(32) NOT NULL,
+  id STRING(36) NOT NULL,
+  A ARRAY<INT64>,
+  B ARRAY<STRING(20)>,
+  C STRING(30),
+  D NUMERIC,
+  E DATE,
+  F TIMESTAMP,
+  G BOOL,
+  H ARRAY<DATE>,
+  I ARRAY<TIMESTAMP>,
```

Comment:
We need tests for other datatypes, Array<BOOL>, Array<BYTES>, Array<FLOAT64>, Array<Int64>, Array<String>, Array<Numeric>, Array<JSON>. Given that we already figure out the test framework, this can be done in a followup pull request.

Reply:
We do have ARRAY<INT64>, ARRAY<STRING>, ARRAY<BOOL>;